modules/oe_content_publication/config/post_updates/00003_create_body_field/field.field.node.oe_publication.body.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
```

Comment:
Please add field.field.node.oe_publication.body.yml to the config/install to create this field on the new sites.